### PR TITLE
Enable 'envFile' for cppdbg

### PR DESCRIPTION
**Pull request by WardenGnaw**
This PR allows the cppdbg configuration to set environment variables
via a envFile.

Example:
ENVIRONMENT_VAR_1=Hello
ENVIRONMENT_VAR_2=15

### CODE REVIEW

**Review by pieandcakes — APPROVED** [review]

File: Extension/package.json [hunk 1]
```diff
@@ -792,6 +792,11 @@
                 },
                 "default": []
               },
+              "envFile": {
+                "type": "string",
+                "description": "Absolute path to a file containing environment variable definitions. These file has key value pairs sepearted by an equals sign per line. E.g. KEY=VALUE",
```
Comment (michelleangela) [inline Extension/package.json]: "This file"? Same text in `OptionsSchema.json`

**Review by michelleangela — APPROVED** [review]

File: Extension/package.json [hunk 2]
```diff
@@ -792,6 +792,11 @@
                 },
                 "default": []
               },
+              "envFile": {
+                "type": "string",
+                "description": "Absolute path to a file containing environment variable definitions. This file has key value pairs sepearted by an equals sign per line. E.g. KEY=VALUE",
```
Comment (sean-mcmanus) [inline Extension/package.json]: spelling

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/package.json (hunk 2 above)
Comment (michelleangela) [reply Extension/package.json]: Should we use something like https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

**Review by sean-mcmanus — APPROVED** [review]

**Comment by polyval** [comment]
@WardenGnaw why ‘envFile’ is not supported at gdb pipe attach configuration, I want to pass some local system environment variables to fill in the configuration. Such as the remote ip、password and program。Since this environment variables are generated after VSCode started, VSCode cannot get these refreshed environment variables unless I restarted it. Do you know any workaround to achieve this.

**Comment by WardenGnaw** [comment]
@polyval Can you make this an issue at https://github.com/microsoft/vscode-cpptools/issues? That would be another feature request.

**Pull request merged**